code/turf/turf.dm

```diff
@@ -124,6 +124,30 @@
 
 	proc/selftilenotify()
 
+	proc/is_frozen()
+		var loc_temp = T0C
+		if(isturf(src.loc))
```

Comment:
the parent of this proc is a turf, so src.loc is always an area. 
also in general this condition isnt necessary because it has to exist, it is itself.